.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - master
```

Comment:
Waiting on admin access to [rename our primary branch](https://github.com/github/renaming) and this to main